### PR TITLE
pangea-node-sdk: add Sanitize docs (GEA-13292)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

**Review by pangea-andrest — APPROVED** [review]

**Review by pangea-andrest — APPROVED** [review]

**Comment by pangea-andrest** [comment]
@kenany should rebase on branch to fix tests

File: packages/pangea-node-sdk/src/services/sanitize.ts [hunk 1]
```diff
@@ -57,7 +98,29 @@ export class SanitizeService extends BaseService {
...
 
-  // TODO: Docs
+  /**
+   * @summary Sanitize via presigned URL
+   * @description Apply file sanitization actions according to specified rules
+   * via a [presigned URL](https://pangea.cloud/docs/api/presigned-urls).
+   * Beta API.
+   * @operationId sanitize_post_v1beta_sanitize 2
+   * @param request Request parameters.
+   * @param options Additional options.
```
Comment (pangea-andrest) [inline packages/pangea-node-sdk/src/services/sanitize.ts:108]: options has been removed

**Review by pangea-andrest — CHANGES_REQUESTED** [review]

File: packages/pangea-node-sdk/src/services/sanitize.ts (hunk 1 above)
Comment (kenany) [reply packages/pangea-node-sdk/src/services/sanitize.ts]: Removed.

**Review by pangea-andrest — APPROVED** [review]

File: packages/pangea-node-sdk/src/services/base.ts [hunk 2]
```diff
@@ -59,7 +59,7 @@ class BaseService {
     return await this.request.downloadFile(url);
   }
 
-  async pollResult(request_id: string): Promise<PangeaResponse<any>> {
+  async pollResult<R>(request_id: string): Promise<PangeaResponse<R>> {
```
Comment (pangea-andrest) [inline packages/pangea-node-sdk/src/services/base.ts:62]: Wait, isn't this a breaking change?

Comment (kenany) [reply packages/pangea-node-sdk/src/services/base.ts]: Now defaulting `R = any` to preserve backwards compat.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**